### PR TITLE
Add blog for OpenSearchCon 2024 NA

### PR DESCRIPTION
**Pull request by prthsh2020**
### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title:  "OpenSearchCon 2024 - North America: The Blog"
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:3]:
```suggestion
title:  "OpenSearchCon 2024: North America -- The blog post"
```

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+meta_description: 
+excerpt: OpenSearch, the community-driven, open-source search and analytics suite, has been gaining momentum in the data-driven application landscape. The recent OpenSearchCon North America 2024 event held in San Francisco, which attracted over 350 attendees from around the world, was a testament to the growing influence of this powerful platform. In this blog post, we'll explore the key highlights, insightful sessions, and exciting announcements that emerged from this year's conference.
+---
+
+
+1. Introduction 
+OpenSearch, the community-driven, open-source search and analytics suite, has been gaining momentum in the data-driven application landscape. The recent ![OpenSearchCon North America 2024](https://opensearch.org/events/opensearchcon/2024/north-america/index.html) event held in San Francisco, which attracted over 350 attendees from around the world, was a testament to the growing influence of this powerful platform. In this blog post, we'll explore the key highlights, insightful sessions, and exciting announcements that emerged from this year's conference.
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:21]:
```suggestion
OpenSearch, the community-driven, open-source search and analytics suite, has been gaining momentum in the data-driven application landscape. The recent ![OpenSearchCon 2024: North America](https://opensearch.org/events/opensearchcon/2024/north-america/index.html) event held in San Francisco, which attracted over 350 attendees from around the world, was a testament to the growing influence of this powerful platform. In this blog post, we'll explore the key highlights, insightful sessions, and exciting announcements that emerged from this year's conference.
```

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+2. Keynote: The Future of OpenSearch
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:23]:
```suggestion
2. Keynote: The future of OpenSearch
```

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+The event kicked off with an inspiring keynote from Eli Fisher, Sr. Manager, Product Management at OpenSearch who laid out the vision for the future of OpenSearch. Some key announcements included:
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:25]:
```suggestion
The event kicked off with an inspiring keynote from Eli Fisher, Sr. Manager, Product Management at OpenSearch, who laid out the vision for the future of OpenSearch. Some key announcements included:
```

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+* OpenSearch 2.x and Unlocking AI potential: Eli shared details about the OpenSearch 2.x release, including features like the OpenSearch AI toolkit, built-in RAG processor, and concurrent segment search, unlocking new possibilities for AI-powered applications.
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:27]:
```suggestion
* OpenSearch 2.x and unlocking AI potential: Eli shared details about the OpenSearch 2.x release, including features like the OpenSearch AI toolkit, a built-in retrieval-augmented generation (RAG) processor, and concurrent segment search, unlocking new possibilities for AI-powered applications.
```

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+* OpenSearch GenAI Use Cases : Anupam Chaturvedi, Adobe Engineering Manager, demonstrated how Adobe is leveraging OpenSearch for its Acrobat GenAI assistant, showcasing the practical applications of OpenSearch in the generative AI space.
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:28]:
```suggestion
* OpenSearch generative AI use cases: Anupam Chaturvedi, Adobe Engineering Manager, demonstrated how Adobe is leveraging OpenSearch for its Acrobat generative AI assistant, showcasing the practical applications of OpenSearch in the generative AI space.
```

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+* Community Growth: Jim Zemlin, Executive Director, ![The Linux Foundation](https://www.linuxfoundation.org/) spoke about welcoming OpenSearch to the Linux Foundation. By opening up the project governance and moving to a neutral home, the OpenSearch team is taking concrete steps to further the project’s founding principle of being for the community and by the community.
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:29]:
```suggestion
* Community growth: Jim Zemlin, Executive Director, ![The Linux Foundation](https://www.linuxfoundation.org/), spoke about welcoming OpenSearch to the Linux Foundation. By opening up project governance and moving to a neutral home, the OpenSearch Project is taking concrete steps to further the project's founding principle of being for the community and by the community. 
```

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+2. Keynote: The Future of OpenSearch
+
+The event kicked off with an inspiring keynote from Eli Fisher, Sr. Manager, Product Management at OpenSearch who laid out the vision for the future of OpenSearch. Some key announcements included:
+
+* OpenSearch 2.x and Unlocking AI potential: Eli shared details about the OpenSearch 2.x release, including features like the OpenSearch AI toolkit, built-in RAG processor, and concurrent segment search, unlocking new possibilities for AI-powered applications.
+* OpenSearch GenAI Use Cases : Anupam Chaturvedi, Adobe Engineering Manager, demonstrated how Adobe is leveraging OpenSearch for its Acrobat GenAI assistant, showcasing the practical applications of OpenSearch in the generative AI space.
+* Community Growth: Jim Zemlin, Executive Director, ![The Linux Foundation](https://www.linuxfoundation.org/) spoke about welcoming OpenSearch to the Linux Foundation. By opening up the project governance and moving to a neutral home, the OpenSearch team is taking concrete steps to further the project’s founding principle of being for the community and by the community. 
+
+3. Important Sessions and Workshops
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:31]:
```suggestion
3. Important sessions and workshops
```

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+The conference featured several informative sessions and workshops, including:
+
+* Session: ![Airbnb Embedding Platform](https://www.youtube.com/watch?v=kpWQ55hVF90)
+    This session, led by Moutupsi Paul, Xiaotang Wang and Amulya Sharma, focused on how Airbnb is leveraging OpenSearch as a vector database for its embedding platform. The session provided an overview of how Airbnb is using vector embeddings to solve guest, host & marketplace challenges, followed by specific use cases that led Airbnb to adopt OpenSearch as its vector database. The session shared lessons learned and provided insights into strategies for optimizing OpenSearch as a vector database.
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:35]:
```suggestion
    This session, led by Moutupsi Paul, Xiaotang Wang, and Amulya Sharma, focused on how Airbnb is leveraging OpenSearch as a vector database for its embedding platform. The session provided an overview of how Airbnb is using vector embeddings to solve guest, host, and marketplace challenges, followed by specific use cases that led Airbnb to adopt OpenSearch as its vector database. The session shared lessons learned and provided insights into strategies for optimizing OpenSearch as a vector database.
```

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+* Session: ![Lucene And Beyond - Core Storage Extension In OpenSearch](https://youtu.be/-_mXJqvPw0o?si=7Lb1Y__uAZ7ZCc66)
+    OpenSearch is tightly bound to the Lucene core APIs that facilitate Encoding, Transactions, Merges, Search. This session, led by Samuel Herman, focused on how the OpenSearch storage encoding can be extended to popular formats (e.g. Parquet, Avro) that are readable by public big data systems such as Apache spark.
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:37]:
```suggestion
    OpenSearch is tightly bound to the Lucene core APIs that facilitate encoding, transactions, merges, and search. This session, led by Samuel Herman, focused on how OpenSearch storage encoding can be extended to popular formats (for example, Parquet, Avro) that are readable by public big data systems such as Apache Spark. 
```

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+    OpenSearch is tightly bound to the Lucene core APIs that facilitate Encoding, Transactions, Merges, Search. This session, led by Samuel Herman, focused on how the OpenSearch storage encoding can be extended to popular formats (e.g. Parquet, Avro) that are readable by public big data systems such as Apache spark. 
+* Session: ![Bringing cloud native innovation from search platforms at Uber and Slack into OpenSearch](https://youtu.be/iZrFVqvTD9s?si=loButiXx8zm2PHob)
+    This session, led by Karthikeyan Ramasamy and Bryan Burkholder, showcased how their organizations’ unique requirements of low latency, high throughput workloads led them to develop their own search and log analytics platforms, making it easy to operate, cost-effective, and scale to petabytes of data. The session covers how Uber and Slack are collaborating with OpenSearch Project to bring some of their learnings and innovations from their platforms into OpenSearch, with special focus on a modular cloud native architecture with isolation between readers and writers and a performant messaging protocol for communication between services.
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:39]:
```suggestion
    This session, led by Karthikeyan Ramasamy and Bryan Burkholder, showcased how their organizations' unique requirements of low-latency, high-throughput workloads led them to develop their own search and log analytics platforms, making it easy to operate, cost effective, and able to scale to petabytes of data. The session covered how Uber and Slack are collaborating with the OpenSearch Project to bring some of their learnings and innovations from their platforms into OpenSearch, with a special focus on a modular cloud-native architecture featuring isolation between readers and writers and a performant messaging protocol for communication between services. 
```

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+* Session: ![Bringing cloud native innovation from search platforms at Uber and Slack into OpenSearch](https://youtu.be/iZrFVqvTD9s?si=loButiXx8zm2PHob)
+    This session, led by Karthikeyan Ramasamy and Bryan Burkholder, showcased how their organizations’ unique requirements of low latency, high throughput workloads led them to develop their own search and log analytics platforms, making it easy to operate, cost-effective, and scale to petabytes of data. The session covers how Uber and Slack are collaborating with OpenSearch Project to bring some of their learnings and innovations from their platforms into OpenSearch, with special focus on a modular cloud native architecture with isolation between readers and writers and a performant messaging protocol for communication between services. 
+* Session: ![A deep dive into OpenSearch Serverless](https://www.youtube.com/watch?v=JPo_uF4Fw4M)
+    This session led by Rohit Nair and Milav Shah, dives deep into how OpenSearch serverless works, including architecture details, understanding performance and operational characteristics and future roadmap.
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:41]:
```suggestion
    This session, led by Rohit Nair and Milav Shah, dove deep into how OpenSearch Serverless works, including architecture details, performance and operational characteristics, and the future roadmap.
```

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:42]: Line 41: "Amazon OpenSearch Serverless"?

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+4. Closing Remarks: What’s Next for OpenSearch?
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:43]:
```suggestion
4. Closing remarks: What's next for OpenSearch?
```

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 15]
```diff
@@ -0,0 +1,51 @@
...
+
+The event was not just about technical sessions—it was also a fantastic opportunity to network with peers in the open-source search space. The community lounge was buzzing with activity, and there were plenty of informal meetups where attendees could connect with OpenSearch engineers, contributors, and advocates.
+
+As the OpenSearchCon event drew to a close, it was clear that the conference was a significant milestone for the OpenSearch community. The announcements around new features, improvements in observability, and the ongoing commitment to open-source principles left attendees excited about the future of the project.
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:47]:
```suggestion
As the OpenSearchCon event drew to a close, it was clear that the conference was a significant milestone for the OpenSearch community. The announcements about new features, improvements in observability, and the ongoing commitment to open-source principles left attendees excited about the future of the project.
```

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 16]
```diff
@@ -0,0 +1,51 @@
...
+
+The OpenSearch roadmap for 2024 highlights key themes around which the contributors are innovating, including stability, availability, and resiliency (StAR); cost, performance, and scale (CoPS); ease of use; observability and log analytics; search and machine learning; security; and a modular architecture.
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:49]:
```suggestion
The OpenSearch roadmap for 2024 highlights key themes around which contributors are innovating, including stability, availability, and resiliency (StAR); cost, performance, and scale (CoPS); ease of use; observability and log analytics; search and machine learning (ML); security; and a modular architecture.
```

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 17]
```diff
@@ -0,0 +1,51 @@
...
+
+As the OpenSearch project continues to grow, the community is streamlining the process of creating a roadmap with ideas and projects that come directly from the community. It's clear that the project is poised to make an even bigger impact in the world of AI/ML, search, and observability. We will be back in 2025 with three different OpenSearchCons, including Europe, India, and North America. You can bookmark the events page and consider attending one of the project's bi-weekly ![Meetups](https://www.meetup.com/opensearch/) or an in-person ![User Groups](https://www.meetup.com/pro/opensearchproject/). We encourage you to try out OpenSearch, engage with the community, and stay tuned for the exciting developments to come.
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:51]:
```suggestion
As the OpenSearch Project continues to grow, the community is streamlining the process of creating a roadmap that includes ideas and projects provided directly by the community. It's clear that the project is poised to make an even bigger impact in the world of AI/ML, search, and observability. We will be back in 2025 with three different OpenSearchCon events, including in Europe, India, and North America. You can bookmark the events page and consider attending one of the project's bi-weekly ![meetups](https://www.meetup.com/opensearch/) or an in-person ![user group](https://www.meetup.com/pro/opensearchproject/). We encourage you to try out OpenSearch, engage with the community, and stay tuned for the exciting developments to come.
```

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 18]
```diff
@@ -0,0 +1,51 @@
...
+
+4. Closing Remarks: What’s Next for OpenSearch?
+
+The event was not just about technical sessions—it was also a fantastic opportunity to network with peers in the open-source search space. The community lounge was buzzing with activity, and there were plenty of informal meetups where attendees could connect with OpenSearch engineers, contributors, and advocates.
+
+As the OpenSearchCon event drew to a close, it was clear that the conference was a significant milestone for the OpenSearch community. The announcements around new features, improvements in observability, and the ongoing commitment to open-source principles left attendees excited about the future of the project.
+
+The OpenSearch roadmap for 2024 highlights key themes around which the contributors are innovating, including stability, availability, and resiliency (StAR); cost, performance, and scale (CoPS); ease of use; observability and log analytics; search and machine learning; security; and a modular architecture.
+
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:50]: Line 51: Should we provide a link to the events page that we invite the reader to bookmark?

**Review by natebower — COMMENTED** [review]
@prthsh2020 @pajuric Editorial review complete. Please see my comments and changes and let me know if you have any questions. Thanks!

**Comment by pajuric** [comment]
@prthsh2020 - Please make the updates to the blog so we are able to publish it on 12/3.

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md (hunk 18 above)
Comment (natebower) [reply _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:50]: @prthsh2020 I think so, but maybe have @pajuric confirm.

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md (hunk 12 above)
Comment (prthsh2020) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md]: This session led by Rohit Nair and Milav Shah, dives deep into how Amazon OpenSearch Serverless works, including architecture details, understanding performance and operational characteristics and future roadmap.

Comment (natebower) [reply _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md]: @prthsh2020 Just need a comma after "session", another after "characteristics", and "the" before "future".

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 19]
```diff
@@ -0,0 +1,51 @@
...
+* Session: ![Bringing cloud native innovation from search platforms at Uber and Slack into OpenSearch](https://youtu.be/iZrFVqvTD9s?si=loButiXx8zm2PHob)
+    This session, led by Karthikeyan Ramasamy and Bryan Burkholder, showcased how their organizations' unique requirements of low-latency, high-throughput workloads led them to develop their own search and log analytics platforms, making it easy to operate, cost effective, and able to scale to petabytes of data. The session covered how Uber and Slack are collaborating with the OpenSearch Project to bring some of their learnings and innovations from their platforms into OpenSearch, with a special focus on a modular cloud-native architecture featuring isolation between readers and writers and a performant messaging protocol for communication between services. 
+* Session: ![A deep dive into OpenSearch Serverless](https://www.youtube.com/watch?v=JPo_uF4Fw4M)
+    This session, led by Rohit Nair and Milav Shah, dove deep into how OpenSearch Serverless works, including architecture details, performance and operational characteristics, and the future roadmap.
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:41]:
```suggestion
    This session, led by Rohit Nair and Milav Shah, dives deep into how Amazon OpenSearch Serverless works, including architecture details, understanding performance and operational characteristics, and the future roadmap.
```

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 20]
```diff
@@ -0,0 +1,51 @@
...
+
+The OpenSearch roadmap for 2024 highlights key themes around which contributors are innovating, including stability, availability, and resiliency (StAR); cost, performance, and scale (CoPS); ease of use; observability and log analytics; search and machine learning (ML); security; and a modular architecture.
+
+As the OpenSearch Project continues to grow, the community is streamlining the process of creating a roadmap that includes ideas and projects provided directly by the community. It's clear that the project is poised to make an even bigger impact in the world of AI/ML, search, and observability. We will be back in 2025 with three different OpenSearchCon events, including in Europe, India, and North America. You can bookmark the events page and consider attending one of the project's bi-weekly ![meetups](https://www.meetup.com/opensearch/) or an in-person ![user group](https://www.meetup.com/pro/opensearchproject/). We encourage you to try out OpenSearch, engage with the community, and stay tuned for the exciting developments to come.
```
Comment (natebower) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:51]:
```suggestion
As the OpenSearch Project continues to grow, the community is streamlining the process of creating a roadmap that includes ideas and projects provided directly by the community. It's clear that the project is poised to make an even bigger impact in the world of AI/ML, search, and observability. We will be back in 2025 with three different OpenSearchCon events, including in Europe, India, and North America. You can bookmark the [Upcoming Events](https://opensearch.org/events/) page and consider attending one of the project's bi-weekly ![meetups](https://www.meetup.com/opensearch/) or an in-person ![user group](https://www.meetup.com/pro/opensearchproject/). We encourage you to try out OpenSearch, engage with the community, and stay tuned for the exciting developments to come.
```

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md (hunk 18 above)
Comment (natebower) [reply _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:50]: Change made

**Comment by prthsh2020** [comment]
> @prthsh2020 - Please make the updates to the blog so we are able to publish it on 12/3.

@pajuric - Done.

File: _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md [hunk 21]
```diff
@@ -0,0 +1,51 @@
...
+ - smelik
+date: 2024-11-14
+categories:
+ - community-updates
+ - community
+ - updates
+ - update
+ - partners
+meta_keywords: OpenSearchCon, OpenSearch Project, Adobe, Linux Foundation, GenAI, Vector Search, OpenSearch Software Foundation, Airbnb, Lucene, Cloud, OpenSearch Serverless, Uber, Slack, OpenSearchCon North America
+meta_description:
```
Comment (pajuric) [inline _posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md:15]: Please update the meta with the following: 

_meta_keywords:_ OpenSearch Project, OpenSearchCon 2024, search and analytics, open-source, AI, machine learning, data-driven applications, cloud-native, community-driven, Linux Foundation

_meta_description:_ Explore the key highlights, insightful sessions, and exciting announcements from the OpenSearchCon North America 2024. Learn about the OpenSearch Project, new features, community growth, and about the roadmap for 2024 and beyond.

**Comment by pajuric** [comment]
@natebower @kolchfa-aws - Can I get your help with pushing this blog live. Thank you.

**Comment by kolchfa-aws** [comment]
@prthsh2020 Please fix DCO so we can merge the PR. Thanks!

**Review by nateynateynate — APPROVED** [review]

**Pull request merged**